src/fragrance_rater/api/imports.py

```
@@ -0,0 +1,68 @@
+
+    total_rows: int = Field(..., description="Total rows processed")
+    imported: int = Field(..., description="Successfully imported count")
+    skipped: int = Field(..., description="Skipped rows count")
+    errors: list[str] = Field(default_factory=list, description="Error messages")
+
+
+@router.post("/kaggle", response_model=ImportResponse)
+async def import_kaggle_csv(
+    file: Annotated[UploadFile, File(description="Kaggle CSV file to import")],
+    session: Annotated[AsyncSession, Depends(get_db)],
+    dry_run: bool = False,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)